source/MaterialXRuntime/RtLogger.h

```diff
@@ -0,0 +1,55 @@
+//
```

Comment:
Fairly simple logging class, can be extended later if we need additional pieces.